src/components/Entry/EntryListFilters/index.tsx

```diff
@@ -8,20 +8,26 @@ import EntryTypeFilter from './EntryTypeFilter';
 import IntegratedFilter from './IntegratedFilter';
 import LatestFilter from './LatestFilter';
 import GOTermsFilter from './GOTermsFilter';
+import AIGeneratedFilter from './AIGeneratedFilter';
 
 type Props = { mainDB?: string };
-export const EntryListFilter = ({ mainDB }: Props) => (
-  <FiltersPanel>
-    <EntryTypeFilter
-      label={`${
-        mainDB === 'InterPro' ? 'InterPro' : 'Member Database Entry'
-      } Type`}
-    />
-    {mainDB !== 'InterPro' && <IntegratedFilter label="InterPro State" />}
-    {mainDB === 'InterPro' && <GOTermsFilter label="GO Terms" />}
-    {mainDB === 'InterPro' && <LatestFilter label="New entries" />}
-  </FiltersPanel>
-);
+export const EntryListFilter = ({ mainDB }: Props) => {
+  const [AILabel, setAILabel] = useState('AI-Generated Entries');
+
+  return (
+    <FiltersPanel>
+      <EntryTypeFilter
+        label={`${
+          mainDB === 'InterPro' ? 'InterPro' : 'Member Database Entry'
+        } Type`}
+      />
+      {mainDB !== 'InterPro' && <IntegratedFilter label="InterPro State" />}
+      {mainDB === 'InterPro' && <GOTermsFilter label="GO Terms" />}
+      {mainDB === 'InterPro' && <LatestFilter label="New entries" />}
+      <AIGeneratedFilter label={AILabel} changeLabel={setAILabel} />
```

Comment:
Unfortunately, changing the label to an empty string only hide the list of options, but there are still an instance of `FilterPanel` being created resulting in an empty space at the bottom:

![image](https://github.com/user-attachments/assets/f66e605d-60f4-430a-bd61-dffe777b5f96)

The simplest is too always display the filter, regardless of the whether there are AI signatures or not.

Reply:
Yeah, I noticed. Unfortunately the way that page is structured seemed to not allow conditional rendering of the filter. Will look better into it to see if there's a better solution.